docs/reference/jspsych-metadata.md

```diff
@@ -0,0 +1,156 @@
+# Metadata Module
```

Comment:
I think what's missing is some kind of clearer narrative-based example that walks through it all. maybe that ends up as a tutorial?